modules/ossm-about-exposing-services-to-traffic-outside-a-cluster.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+
+// gateways/ossm-about-gateways.adoc
+
+:_mod-docs-content-type: Concept
+[id="ossm-about-exposing-services-to-traffic-outside-a-cluster_{context}"]
+= About exposing services to traffic outside a cluster
+:context: ossm-about-exposing-services-to-traffic-outside-a-cluster
+
+To enable traffic from outside an {ocp-short-name} cluster to access services in a mesh, you must expose a gateway proxy by either setting its `Service` type to `LoadBalancer` or by using the {ocp-short-name} router.
+
+Using Kubernetes load balancing to handle incoming traffic directly through the inbound gateway can reduce latency associated with data encryption. By managing encryption at the inbound gateway, you avoid the intermediate decryption and re-encryption steps within the mesh that often add latency. This approach allows mesh traffic to be encrypted and decrypted only once, which is generally more efficient.
+
+The {ocp-short-name} router provides a standard approach for managing ingress traffic, and enables you to manage certificates for all cluster ingress using the same methods. However, the  {ocp-short-name} router introduces an additional hop between the inbound traffic and the mesh applications. Typically, you route the traffic by decrypting it at the router and then re-encrypting it at the service mesh ingress gateway, which introduces latency.
+
+// If you are using the Gateway API with automatic deployment, this is the default setting. If you are using Gateway Injection, you have to set the field manually.
```

Comment:
@dgn any idea why we are not capturing docs about Gateway API?

Reply:
I think Jamie doesn't want us to cover it until we can ship the CRDs in a GA way (which most likely means when OCP ships them)

Reply:
Correct, we were told to wait for Gateway API docs because CRDs are not supported at this time.